system.h

```diff
@@ -92,7 +90,7 @@
 // Define global system variables
 typedef struct {
   uint8_t abort;                 // System abort flag. Forces exit back to main loop for reset.
-  uint8_t state;                 // Tracks the current state of Grbl.
+  uint16_t state;                 // Tracks the current state of Grbl.
```

Comment:
Can you do me a favor and grep around to make sure that we're not relying on an 8 bit state variable anywhere?  I'd hate for higher order bits to get truncated accidentally due to an eager cast.